packages/expo-router/src/link/elements.tsx

```diff
@@ -0,0 +1,249 @@
+   *
+   * https://developer.apple.com/documentation/uikit/uimenuelement/attributes/disabled
+   */
+  disabled?: boolean;
+  /**
+   * If true, the menu item will be displayed as destructive.
+   *
+   * https://developer.apple.com/documentation/uikit/uimenuelement/attributes/destructive
+   */
+  destructive?: boolean;
+  /**
+   * If true, the menu will be kept presented after the action is selected.
+   *
+   * This is marked as unstable, because when action is selected it will recreate the menu,
+   * which will close all opened submenus and reset the scroll position.
+   *
+   * https://developer.apple.com/documentation/uikit/uimenuelement/attributes/keepsmenupresented
+   */
+  unstable_keepPresented?: boolean;
+  /**
+   * If true, the menu item will be displayed as selected.
+   */
+  isOn?: boolean;
+  onPress: () => void;
+}
+
+/**
+ * This component renders a context menu action for a link.
+ * It should only be used as a child of `Link.Menu` or `LinkMenu`.
+ *
+ * **Note:** You can use the alias `Link.MenuAction` for this component.
+ *
+ * @platform ios
+ */
+export function LinkMenuAction(props: LinkMenuActionProps) {
+  if (useIsPreview() || process.env.EXPO_OS !== 'ios' || !use(InternalLinkPreviewContext)) {
+    return null;
+  }
+  const { unstable_keepPresented, onPress, ...rest } = props;
+  return (
+    <NativeLinkPreviewAction
+      {...rest}
+      onSelected={onPress}
+      keepPresented={unstable_keepPresented}
+    />
+  );
+}
+
+export interface LinkMenuProps {
+  /**
+   * The title of the menu item
+   */
+  title?: string;
+  /**
+   * Optional SF Symbol displayed alongside the menu item.
+   */
+  icon?: string;
+  /**
+   * If true, the menu will be displayed as a palette.
+   * This means that the menu will be displayed as one row
+   *
+   * https://developer.apple.com/documentation/uikit/uimenu/options-swift.struct/displayaspalette
+   */
+  displayAsPalette?: boolean;
+  /**
+   * If true, the menu will be displayed inline.
+   * This means that the menu will not be collapsed
+   *
+   * https://developer.apple.com/documentation/uikit/uimenu/options-swift.struct/displayinline
+   */
+  displayInline?: boolean;
+  /**
+   * If true, the menu item will be displayed as destructive.
+   *
+   * https://developer.apple.com/documentation/uikit/uimenu/options-swift.struct/destructive
+   */
+  destructive?: boolean;
+  children: ReactElement<LinkMenuActionProps> | ReactElement<LinkMenuActionProps>[];
+}
+
+/**
+ * Groups context menu actions for a link.
+ *
+ * If multiple `Link.Menu` components are used within a single `Link`, only the first will be rendered.
+ * Only `Link.MenuAction` and `LinkMenuAction` components are allowed as children.
+ *
+ * @example
+ * ```tsx
+ * <Link.Menu>
+ *   <Link.MenuAction title="Action 1" onPress={() => {}} />
+ *   <Link.MenuAction title="Action 2" onPress={() => {}} />
+ * </Link.Menu>
+ * ```
+ *
+ * **Note:** You can use the alias `Link.Menu` for this component.
```

Comment:
```suggestion
 * > **Note**: You can use the alias `Link.Menu` for this component.
```

Let's use the default callout type for the note.